lib/rubocop/cop/rails/file_path.rb

```diff
@@ -34,7 +37,7 @@ module Rails
       #   # good
       #   Rails.root.join('app', 'models', 'goober').to_s
       #
-      class FilePath < Base
+      class FilePath < Base # rubocop:disable Metrics/ClassLength
```

Comment:
I'm not sure if this cop should be permanently disabled with `rubocop:disable Metrics/ClassLength`. Can you add `Include` entry to `.rubocop_todo.yml` instead?

```yaml
Metrics/ClassLength:
  Max: 163
  Exclude:
    - 'lib/rubocop/cop/rails/file_path.rb'
```

Reply:
Okay, I've fixed it, but I had seen the same kind of exception for `Rails/RootPathnameMethods`.

https://github.com/rubocop/rubocop-rails/blob/master/lib/rubocop/cop/rails/root_pathname_methods.rb#L38

Reply:
That case involves many multi-line arrays, so the context may be slightly different from this one. In any case, the existing directive doesn't require changes for now. Let's leave it as is -)